pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,24 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        data_df = pd.DataFrame([{'col_a': '1', 'col_b': '16.5%',
```

Comment:
Can you add the GH issue number as a comment?

Reply:
Added in 1c35cc70108a0407d964c26b5cfc07d654dd4ebc